react-components/src/components/CacheProvider/RevisionFdmNodeCache.ts

```diff
@@ -247,9 +251,13 @@ export class RevisionFdmNodeCache {
 
       return edgesAndNodes.map((edge) => edge.edge);
     }
+    const modelInstances = await this._model;
```

Comment:
Can we have getters and setters instead of accessing internal/private parameters? Even internally I think it would be better to use it, unless we dont follow this OOP pattern or it has some consequences that I'm not aware of in our case.

Reply:
getter and setter is useful only when it is used outside this class and we access it else where

Reply:
Actually, in some cases getter and setter are also good to be used internally, mostly for future code, single point of truth, future simplicity, etc.. is a common OOP pattern.
Like if we have to introduce some extra check before return/set that object then we have the opportunity to do that in one single place (inside the getter or setter) without replicating the same code all over other places that needs to have that treatment. Then we're ending up creating a function for that, which behaves like a getter/setter in the end of the day.

But ofc, I dont have a strong opinion in our case, as we're using this pattern already.